cmd/autoheal/alerts_worker.go

```diff
@@ -88,9 +88,17 @@ func (h *Healer) startHealing(alert *alertmanager.Alert) error {
 	activated := make([]*monitoring.HealingRule, 0)
 	h.rulesCache.Range(func(_, value interface{}) bool {
 		rule := value.(*monitoring.HealingRule)
-		if h.checkConditions(rule, alert) {
+		ok, err := h.checkRule(rule, alert)
```

Comment:
ok usually means something can [succeed or fail]](https://tour.golang.org/methods/15)
 Maybe `matches`?

Reply:
Yes, `matches` makes sense, will change it.